project/PrivateProject.scala

```diff
@@ -5,15 +5,27 @@ import com.typesafe.tools.mima.plugin.MimaPlugin
 import com.typesafe.tools.mima.plugin.MimaPlugin.autoImport._
 import explicitdeps.ExplicitDepsPlugin.autoImport._
 import sbt._
-import verizon.build.DisablePublishingPlugin
+import Keys._
 
 object PrivateProjectPlugin extends AutoPlugin {
   override def trigger = noTrigger
 
-  override def requires = Http4sPlugin && MimaPlugin 
+  override def requires = Http4sPlugin && MimaPlugin
 
   override lazy val projectSettings: Seq[Setting[_]] =
-    DisablePublishingPlugin.projectSettings ++ Seq(
+    Seq(
+      publishLocal := {},
+      publish := {},
+      publishArtifact in Test := false,
+      publishArtifact in Compile := false,
+      publishArtifact in (Test, packageBin) := false,
+      publishArtifact in (Test, packageDoc) := false,
+      publishArtifact in (Test, packageSrc) := false,
+      publishArtifact in (Compile, packageBin) := false,
+      publishArtifact in (Compile, packageDoc) := false,
+      publishArtifact in (Compile, packageSrc) := false,
+      publishArtifact := false
+    ) ++ Seq(
```

Comment:
Is `skip in publish := true` sufficient in modern sbt?

Reply:
I can port in my personal anti publishing subset as well.

Reply:
Should these use the newer slash-syntax instead? I like that a lot better.

Ie. `packageSrc / Compile / publishArtifact := false`

Reply:
I could not get this to work, we can adopt it in a future iteration if you can get it running. 😄